server/neptune/workflows/internal/terraform/job/step/init/runner.go

```diff
@@ -0,0 +1,33 @@
+package init
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/activities"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/job"
+	"github.com/runatlantis/atlantis/server/neptune/workflows/internal/root"
+	"go.temporal.io/sdk/workflow"
+)
+
+type initActivities interface {
+	TerraformInit(ctx context.Context, request activities.TerraformInitRequest) (activities.TerraformInitResponse, error)
+}
+
+type Runner struct {
+	Activity initActivities
+}
+
+func (r *Runner) Run(executionContext *job.ExecutionContext, localRoot *root.LocalRoot, step job.Step) (string, error) {
```

Comment:
feels a bit overkill to keep it in its own package don't you think? Should we just keep this in the job package? (Also feels like it doesn't need to be a struct and can just be another function imo)

Reply:
Do you meant just for init or the other steps too? Yeah I agree it looks like an overkill. I originally organized it like this assuming we'd have to do some deterministic work before orchestrating the activity.

Reply:
doesn't seem like there's any thing else going on, let's just leave it in the job package